tmva/sofie/README.md

```diff
@@ -15,33 +15,33 @@ This is a new development in TMVA and is currently in early experimental stage.
 
 Build ROOT with the cmake option tmva-sofie enabled.
 
-    $ cmake ../root -Dtmva-sofie=ON
-    $ make -j8
+    `$ cmake ../root -Dtmva-sofie=ON`
+    `$ make -j8`
 
 
 ## Usage
 
 
 From ROOT command line, or in a ROOT macro:
 
-	using namespace TMVA::Experimental;
-	SOFIE::RModelParser_ONNX parser;
-	SOFIE::RModel model = parser.Parse(“./example_model.onnx”);
-	model.Generate();
-	model.OutputGenerated(“./example_output.hxx”);
+    `using namespace TMVA::Experimental;`
```

Comment:
You don't need to add these quotes. Doxygen understands that is formatted text (see https://github.com/root-project/root/tree/master/tmva/sofie)

Please remove them

Reply:
I did this because of:
`/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/sofie/README.md:45: warning: explicit link request to 'include' could not be resolved`
https://lcgapp-services.cern.ch/root-jenkins/view/ROOT/job/root-makedoc-master/2590/consoleText

but maybe only line 45 is needed ?

Reply:
Removed.

Reply:
I left the one in the include.
The problem here is that what works well for 'Markdown' format does not work well when generating the doxygen HTML documentation. That's why the escape is needed. Otherwise, we should use native C++ doxygen code blocks instead of Markdown ones which are not fully supported by doxygen.